docs/mcp/run-python.md

```diff
@@ -0,0 +1,146 @@
+# MCP Run Python
+
+The **MCP Run Python** package is an MCP server that allows agents to execute Python code in a secure, sandboxed environment. It uses [Pyodide](https://pyodide.org/) to run Python code in a JavaScript environment, isolating execution from the host system.
+
+## Features
+
+* **Secure Execution**: Run Python code in a sandboxed WebAssembly environment
+* **Package Management**: Automatically detects and installs required dependencies
+* **Complete Results**: Captures standard output, standard error, and return values
+* **Asynchronous Support**: Runs async code properly
+* **Error Handling**: Provides detailed error reports for debugging
+
+## Installation
+
+The MCP Run Python server is distributed as an [NPM package](https://www.npmjs.com/package/@pydantic/mcp-run-python) and can be run directly using [`npx`](https://docs.npmjs.com/cli/v8/commands/npx):
+
+```bash
+npx @pydantic/mcp-run-python [stdio|sse]
+```
+
+Where:
+
+* `stdio`: Runs the server with [stdin/stdout transport](https://modelcontextprotocol.io/docs/concepts/transports#standard-input%2Foutput-stdio) (for subprocess usage)
+* `sse`: Runs the server with [HTTP Server-Sent Events transport](https://modelcontextprotocol.io/docs/concepts/transports#server-sent-events-sse) (for remote connections)
+
+Usage of `@pydantic/mcp-run-python` with PydanticAI is described in the [client](client.md#mcp-stdio-server) documentation.
+
+## Direct Usage
+
+As well as using this server with PydanticAI, it can be connected to other MCP clients, for clarity in this example we connect directly using the [Python MCP client](https://github.com/modelcontextprotocol/python-sdk).
```

Comment:
```suggestion
As well as using this server with PydanticAI, it can be connected to other MCP clients. For clarity, in this example we connect directly using the [Python MCP client](https://github.com/modelcontextprotocol/python-sdk).
```